outlines/generate/generator.py

```diff
@@ -1,9 +1,8 @@
 import dataclasses
 import math
 from typing import TYPE_CHECKING, Callable, Iterator, List, Optional, Tuple, Union
-
+from outlines.caching import diskcache, diskcache_memory, hash_it
```

Comment:
It seems that not all these imports are used.

Reply:
Removed.